script/install.sh

```diff
@@ -417,6 +423,10 @@ install_compiler() {
             local arch="zip"
         fi
 
+        if [[ $ver > "2.072.0z" ]]; then
```

Comment:
Hmm on a second though: this seems to exclude `2.072.0`, should I change it to sth. like `2.071.9`?